activemodel/lib/active_model/errors.rb

```diff
@@ -81,6 +81,18 @@ def initialize_dup(other) # :nodoc:
       super
     end
 
+    # Copies the errors from <tt>other</tt>, if <tt>other</tt> is an ActiveModel::Errors.
+    #
+    #   person.errors.copy!(other)
+    def copy!(other) # :nodoc:
+      if other.is_a?(ActiveModel::Errors)
+        @messages = other.messages.dup
+        @details  = other.details.dup
+      else
+        raise ArgumentError, "#{other.inspect} is not an ActiveModel::Errors"
```

Comment:
:+1:

I found that a lot of `raise ArgumentError, ...` and did the same.
For me `just call the other.messages.dup and other.details.dup` looks better.